libs/cart/testing/src/drivers/in-memory/cart-item/cart-item.service.spec.ts

```diff
@@ -0,0 +1,154 @@
+import { TestBed } from '@angular/core/testing';
+import { HttpClientTestingModule, HttpTestingController } from '@angular/common/http/testing';
+
+import { DaffCart, DaffCartItem } from '@daffodil/cart';
+
+import { DaffCartFactory } from '../../../factories/cart.factory';
+import { DaffInMemoryCartItemService } from './cart-item.service';
+import { DaffCartItemFactory } from '@daffodil/cart/testing';
+import { DaffCartItemInput } from 'dist/cart/models/cart-item-input';
+
+describe('Driver | In Memory | Cart | CartItemService', () => {
```

Comment:
I think these tests are a little misleading, and I'm guessing we probably do it in the other in memory drivers. But really, the only thing this should test is that calling a method starts a request. The part about actually returning what is supposed to be returned isn't actually done here. So I think the tests should just be like:
```
describe('list', () => {
    it('should send a get request for a list of cart items', done => {
      cartItemService.list(cartId).subscribe(res => {
        done();
      });

      const req = httpMock.expectOne(`${cartItemService.url}/${cartId}/`);

      expect(req.request.method).toBe('GET');
      req.flush();                           <------I'm not sure whether you'll need to flush with actual data or not.
    });
  });
```

Reply:
Changed